### DISCUSSION TITLE
Installing from source for Windows 11?

### DISCUSSION THREAD

**Discussion by ultrasquid** [fontforge/fontforge#5511] [General]
The existing Install.md only includes instructions for installing Fontforge from source for Ubuntu and Mac OS. There are no instructions for any version of Windows or other operating systems.

It has been nearly 2 years since the last official release of installer packages. A great deal has changed in Fontforge and its libraries since then, including a debug of libSpiro that should resolve the broken feature of the 2023 FF release. 
I would like to use the updated changes on my Windows device, but I don't know how to do that without a precompiled installer.

**Comment by iorsh** [comment]
It's somewhat buried down, but if you click the check mark next to the latest commit, it would lead you to actions and artifacts, one of which is a Windows 32/64 installer.
![image](https://github.com/user-attachments/assets/b4a4ddad-91c9-4891-b620-fdb832633a13)
After clicking the check mark go to "Details" -> "Summary", scroll to the end of page, pick installer from "Artifacts"

**Reply by iorsh** [reply]
MINGW* is for Windows

**Reply by ultrasquid** [reply]
I see what that checkmark link leads to, but I will need some hand-holding to actually put it to use.

There should at least be a link to that page from the Install.md document

**Reply by iorsh** [reply]
These artifacts are purged in a few months, so links wouldn't really do.
